projects/client/src/app/components/footer/Footer.tsx

```diff
@@ -23,43 +45,81 @@ function Footer() {
 
         <div className='mb-4 flex flex-col items-start lg:mb-0 lg:items-start'>
           <p style={{ fontWeight: 'bold' }}>Useful Links</p>
-          <Button className='text-slate-500'>About</Button>
-          <Button className='text-slate-500'>Contact</Button>
-          <Button className='text-slate-500'>Wishlist</Button>
-          <Button className='text-slate-500'>Terms & Conditions</Button>
+          <Button
+            onClick={() => {
+              navigate('/about  ');
+            }}
+            className='text-slate-500'
+          >
+            About
+          </Button>
+          <Button
+            onClick={() => {
+              navigate('/contact  ');
+            }}
+            className='text-slate-500'
+          >
+            Contact
+          </Button>
+          <Button
+            onClick={() => {
+              navigate('/wishList  ');
+            }}
+            className='text-slate-500'
+          >
+            Wish List
+          </Button>
+          <Button
+            onClick={() => {
+              navigate('/termsConditions  ');
+            }}
+            className='text-slate-500'
+          >
+            Terms & Conditions
+          </Button>
         </div>
 
         <div className='flex flex-col items-start lg:items-start'>
           <p style={{ fontWeight: 'bold' }}>Customer Service</p>
-          <Button className='text-slate-500'>My Account</Button>
-          <Button className='text-slate-500'>My Cart</Button>
+          <Button
+            onClick={() => {
```

Comment:
if it is one line on the onclick function is not needed the {} (same comment as above)